src/library_async.js

```diff
@@ -39,14 +39,14 @@ mergeInto(LibraryManager.library, {
       dbg('asyncify instrumenting imports');
 #endif
       var ASYNCIFY_IMPORTS = {{{ JSON.stringify(ASYNCIFY_IMPORTS.map((x) => x.split('.')[1])) }}};
+      var importPatterns = ASYNCIFY_IMPORTS.map(i => new RegExp(`^${i.replaceAll('*', '.*')}$`));
```

Comment:
We should be able to do all this a build time I think (i.e. inside the `{{{` `}}}`).

Can we combine with the line above?  (what is the line above even doing with the `split('.')[1]`?)

Reply:
Oh I see its stripping of the `env.` prefixes.